src/hydra-evaluator/hydra-evaluator.cc

```diff
@@ -13,7 +13,57 @@
 
 using namespace nix;
 
-typedef std::pair<std::string, std::string> JobsetName;
+typedef std::pair<std::string, std::string> JobsetSymbolicIdentity;
+
+class JobsetIdentity {
+    public:
+
+    std::string project;
+    std::string jobset;
+    int id;
+
+
+    JobsetIdentity(const std::string& project, const std::string& jobset, const int id)
+        : project{ project }, jobset{ jobset }, id{ id }
+    {
+    }
+
+    friend bool operator== (const JobsetIdentity &lhs, const JobsetIdentity &rhs);
```

Comment:
```suggestion
    friend bool operator== (const JobsetIdentity & lhs, const JobsetIdentity & rhs);
```

The style in Nix and Hydra is to have a space on both sides of `&` and `*`.

Reply:
:+1: